be/src/olap/reader.cpp

```diff
@@ -237,6 +239,7 @@ Status TabletReader::_capture_rs_readers(const ReaderParams& read_params) {
     _reader_context.read_orderby_key_columns =
             _orderby_key_columns.size() > 0 ? &_orderby_key_columns : nullptr;
     _reader_context.predicates = &_col_predicates;
+    _reader_context.late_arrival_predicates = &_late_arrival_predicates;
```

Comment:
This method will be called at the beginning of the scanner, If new predicates arrived, is it will affect??